api/lib/source/polkadotV0-source.js

```diff
@@ -620,132 +618,51 @@ class polkadotAPI {
     )
   }
 
-  constructProposal(api, bytes) {
-    let proposal
-
-    try {
-      proposal = api.registry.createType('Proposal', bytes.toU8a(true))
-    } catch (error) {
-      console.log(error)
-    }
-
-    return proposal
-  }
-
-  async getDemocracyProposalMetadata(
-    proposal,
-    description,
-    proposer,
-    proposalMethod,
-    creationTime
-  ) {
+  async getDemocracyProposalMetadata(proposal) {
     const api = await this.getAPI()
 
-    description = `This is a Democracy Proposal whose description and title have not yet been edited on-chain. Only the proposer address (${
-      proposal.proposer || proposer
-    }) is able to change it.`
+    let creationTime
+    let proposer = { name: '', address: '' }
+    let description = `This is a Democracy Proposal whose description and title have not yet been edited on-chain. Only the proposer address (${proposal.proposer}) is able to change it.`
```

Comment:
```suggestion
    let description = ``
```

Reply:
we are using the summary field for explanations and the description field for the core details from the on-chain proposals

Reply:
ah, right